.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Créer un rapport de bogue
+title: ''
+labels: Triage
+assignees: ''
+
+---
+
+**Décrire le bogue**
+Une description claire et concise de ce qu'est le bogue.
+
+**Reproduire**
+Etapes pour reproduire le comportement :
+
+1. Allez à '...'
+2. Cliquez sur '....'
+3. Faites défiler l'écran jusqu'à '....'
+4. Voir l'erreur
+
+**Comportement attendu**
+Une description claire et concise de ce que vous attendez.
+
+**Captures d'écran**
+Le cas échéant, ajoutez des captures d'écran pour expliquer votre problème.
+
+**Environnement (veuillez compléter les informations suivantes) :**
+
+- Navigateur [par ex. chrome, safari]
+- Version du navigateur (optionel) [par ex. 100]
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
- Version du navigateur (optionnel) [par ex. 100]
```